src/CircuitBreaker/src/Hystrix.MetricsStreamCore/RabbitMetricsStreamPublisher.cs

```diff
@@ -32,7 +32,7 @@ public RabbitMetricsStreamPublisher(IOptions<HystrixMetricsStreamOptions> option
             if (sslOption != null && sslOption.Enabled)
             {
                 logger?.LogInformation("Hystrix Metrics using TLS");
-                sslOption.Version = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+                sslOption.Version = SslProtocols.Tls12 | SslProtocols.Tls13;
```

Comment:
Tests aren't likely to notice this change, I'm not sure about real-world impact but everything less than TLS 1.2 has been deprecated